android/src/main/kotlin/top/kikt/imagescanner/core/entity/PermissionResult.kt

```diff
@@ -0,0 +1,7 @@
+package top.kikt.imagescanner.core.entity
+
+enum class PermissionResult(val value: Int) {
```

Comment:
So we don't have `Limited` in this PR?

Reply:
@AlexV525 isn't `limited` an iOS only feature? This is in the Android plugin code, isn't it?

Reply:
> This is in the Android plugin code, isn't it?

Yes you're right. 👍